webtau-http/src/main/java/com/twosigma/webtau/http/Http.java

```diff
@@ -964,4 +981,16 @@ private Object extractOriginalValue(Object v) {
     private interface HttpCall {
         HttpResponse execute(String fullUrl, HttpHeader fullHeader);
     }
+
+    private static class BeforeFirstHttpCallListenerTrigger {
+        static {
+            HttpListeners.beforeFirstHttpCall();
```

Comment:
Why not put this in a static initialiser block for the `Http` class?  Alternatively, `Http` is essentially a singleton, we could make its constructor private and call this function in there.

Reply:
One can define`http.query` or `http.header` ahead of time and that will trigger a callback. Wanted to be on a more restricted side.

Reply:
That would technically still be before the 1st http call 😄 but I see what you're getting at.

Reply:
That may not be the case of JUnit like runners. It may load class, trigger its static inits including `http.header`, but then won't run any `http.get` like calls.